frontend/src/Pages/Login Page/LoginPage.jsx

```diff
@@ -16,24 +16,34 @@ function LoginPage () {
   function handleLogin (event) {
     event.preventDefault()
     setError('')
-    // check to see if both email and password were entered
+    // check to see if both username and password were entered
     // return error message if either were missing
     if (username !== '' && password !== '') {
-      // check if email and password are valid w/ db
-      // if valid redirect to dashboard
-      // return error message if incorrect email or password
-      // const user = {
-      //   username: username,
-      //   password: password
-      // } // send to backend
-      if(1) {  
-        // console.log(email, password)
-        // console.log("redirect to dashboard")
-        navigate('/dashboard')
-      } else {
-        setError('incorrect username or password')
-      }
-    } else if (username === '' && password !== '') {
+      const loginInfo = {
+        username: username,
+        password: password
+      } 
+      // send username and password to login func in apiClient
+      // apiClient sends a req to backend login endpoint to check if username and password matches an existing user in the db
+      // if it matches, user is navigated to dashboard
+      apiClient.login(loginInfo).then( res => {
+        // console.log('data ', res)
+        if(res.data.success) { 
+          console.log('userId ', res.data.userId)
+          console.log('username ', res.data.username)
```

Comment:
Looks Great! Once auth context PR is merged, just make a quick hot fix to save the info into the user object :)